lib/ansible/modules/cloud/amazon/s3_bucket.py

```diff
@@ -112,7 +112,7 @@
 
 import os
 import xml.etree.ElementTree as ET
-import urlparse
+import ansible.module_utils.six.moves.urllib_parse as urlparse
```

Comment:
From the six documentation, it looks as though the official way to do this is 
```
import ansible.module_utils.six.moves.urllib.parse as urlparse
```

Let's do it that way just to be more future-proof.